### PR TITLE
increase timeout for link checking

### PR DESCRIPTION
**Pull request by bgruening**
+ fix a bunch of linting issues.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ build: ## build files but do not run a server
 .PHONY: build
 
 check: build ## validate HTML
-	bundle exec htmlproofer --http-status-ignore 405,999 --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/" --file-ignore "/.*\/files\/.*/" ./_site
+	timeout 120s bundle exec htmlproofer --http-status-ignore 405,999 --url-ignore "/.*localhost.*/","/.*vimeo\.com.*/" --file-ignore "/.*\/files\/.*/" ./_site
```
Comment (bebatut) [inline Makefile]: We will probably need also to add `timeout 120s` in the `.travis.yml` for the `gh-page` branch

Comment (bgruening) [reply Makefile]: Why do we run this test on this branch at all?

Comment (shiltemann) [reply Makefile]: I set up nightly tests on the branch so that we are warned if links break on the site

Comment (bgruening) [reply Makefile]: Ha, nice!

**Pull request merged**